tests/build_/test_plugins_configuration.py

```diff
@@ -305,18 +308,19 @@ def test_render_prod_not_flatpak(self, build_type):
 
         plugins = get_plugins_from_build_json(build_json)
 
-        with pytest.raises(NoSuchPluginException):
-            get_plugin(plugins, "prebuild_plugins", "resolve_module_compose")
-        with pytest.raises(NoSuchPluginException):
-            get_plugin(plugins, "prebuild_plugins", "flatpak_create_dockerfile")
-        with pytest.raises(NoSuchPluginException):
-            get_plugin(plugins, "prepublish_plugins", "flatpak_create_oci")
-        if build_type == BUILD_TYPE_ORCHESTRATOR:
+        if not flatpak:
```

Comment:
What's the point of this test when `flatpak` is true? It doesn't seem to do much and there are no assertions in such case.

Reply:
I think it needs to be replaced with if flatpak and flatpak_base_image and some code to assert that the flatpak stuff is there, and then the else code that has this stuff in it.

It would be best to just squash both test_render_flatpak and test_render_prod_not_flatpak into one test.

Reply:
So I have added the assert for the `flatpak and flatpak_base_image` case. I ll look at `test_render_flatpak`